reana_workflow_controller/rest/workflows_status.py

```diff
@@ -278,6 +283,9 @@ def get_workflow_status(workflow_id_or_name):  # noqa
 
     try:
         user_uuid = request.args["user"]
+        include_last_command = json.loads(
+            request.args.get("include_last_command", "false").lower()
+        )
```

Comment:
What about validating and parsing parameters with `use_args`, as done in the other endpoint?